apps/api/pages/api/credential-sync/_post.ts

```diff
@@ -88,26 +88,29 @@ async function handler(req: NextApiRequest) {
 
   const appMetadata = appStoreMetadata[app.dirName as keyof typeof appStoreMetadata];
 
-  const credential = await prisma.credential.create({
+  const createdcredential = await prisma.credential.create({
     data: {
       userId,
       appId: appSlug,
       key,
       type: appMetadata.type,
     },
-    include: {
-      user: {
-        select: {
-          email: true,
-        },
-      },
+    select: credentialForCalendarServiceSelect,
+  });
+  // createdcredential.user.email;
+  // TODO:              ^ Investigate why this select doesn't work.
```

Comment:
@joeauyeung  Found some odd behavior in here. Seems like using a select or include in here doesn't actually returns the correct types. Tried replicating in other files and on a new repo with no luck. Also tried upgrading prisma to discard a possible bug but still didn't work. I had to create and find in two queries as a workaround.